dev-python/parallax/parallax-1.0.6.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
I do not know why this is necessary, but if you work on a different ebuild, you should put that in a separate commit e.g. `dev-python/parallax: revbump, update PYTHON_COMPAT to python3.10`
That said, couldn't you bump that to 3.11 instead of only 10?
Also, when you work on an ebuild, please copy it and rename it to a new revision, like `paralax-1.0.6-r1.ebuild`. Then make all keywords to unstable, e.g. `~amd64`

Reply:
crmsh needs python 3.10 with parallax for 3.10. why not 3.11, good question, maybe because i'm not experienced enough

Reply:
renamed it to -r1